UnitTests/Tests/WeatherUpdateSpec.swift

```diff
@@ -0,0 +1,61 @@
+@testable import Tropos
+import CoreLocation
+import Quick
+import Nimble
+
+private func weatherConditions(temperature temperature: Int = 90, precipitationProbability: String? = nil, precipitationType: String? = "rain") -> [String: AnyObject] {
+    var data: [String: AnyObject] = [
+        "temperatureMin": 50,
+        "temperatureMax": 60,
+    ]
+
+    if let probability = precipitationProbability {
+        data["precipProbability"] = probability
+        if let type = precipitationType {
+            data["precipType"] = type
+        }
+    }
+
+    return [
+        "currently": ["temperature": temperature],
+        "daily": ["data": [data]],
+    ]
+}
+
+final class WeatherUpdateSpec: QuickSpec {
+    override func spec() {
+        describe("TRWeatherUpdate") {
+            context("currentTemp is higher than currentHigh") {
+                it("updates currentHigh to match") {
+                    let conditions = weatherConditions(temperature: 70)
```

Comment:
Have you considered placing line breaks between [arrange, act and assert] phase?

[arrange, act and assert]: http://c2.com/cgi/wiki?ArrangeActAssert

I would put line breaks between the three phases of these tests: setup, exercise, and verification. As these tests grow, it will be more important to clearly delineate what's happening where.
 Add a line note